libdnf/swdb/item.hpp

```diff
@@ -26,18 +26,42 @@
 
 #include "libdnf/utils/sqlite3/sqlite3.hpp"
 
+/// Class representing a package.
+/**
+ * Here goes the detailed description.
+ * It can span multiple lines in the comment block.
+ *
+ * Paragraphs are delimited with blank lines.
+ * Markdown is supported.
+ */
 class Item {
 public:
-    Item(std::shared_ptr<SQLite3> conn);
+    /// Default contructor.
+    /** A more elaborate description of the constructor. */
```

Comment:
I don't think this kind of comments bring any useful info to user. You can see that it is constructor and as it is only one it is default.

Reply:
This is just an example how docstrings should look (format). Don't worry, there will be fixup soon.